pkg/image/manifest_test.go

```diff
@@ -40,21 +40,36 @@ func TestGetDefaultImageRegistryVersionValidation(t *testing.T) {
 		{
 			name:    "v1.13 is valid",
```

Comment:
I would update the names of these tests to `X is not valid`.

Reply:
Oh ok got it.